.travis.yml

```diff
@@ -19,9 +19,6 @@ matrix:
     env: DEPLOY=true
   - jdk: openjdk14
   - jdk: openjdk-ea
-  allow_failures:
-  - jdk: openjdk14
-  - jdk: openjdk-ea
```

Comment:
We still allow failures for openjdk-ea, which is a floating release in the sense that it is always the actual JDK development and not tied to a specific version.

Reply:
You are right Daniel. I hesitated when I deleted this line. I thought it would be easier to detect failure because when the EA version is changed and introduce new warnings or errors, the build status will be :x: directly. But your argument makes more sense. I added back the lines.

Reply:
I see you point! Thank you, I will merge it